recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "3.0.11" %}
+{% set version = "3.0.11-1" %}
```

Comment:
There are [no changes between these tags]( https://github.com/cython/cython/compare/3.0.11-1...3.0.11 ). Also no corresponding release on PyPI

Think we can safely ignore this

Reply:
Filing issue ( https://github.com/conda-forge/cython-feedstock/issues/141 ) to have the bot ignore this